File: 2026-08/src/day_28/prms_120850/Solution.java
package day_28.prms_120850;

import java.util.*;

class Solution {
    public int[] solution(String my_string) {
        List<Integer> list = new ArrayList<>();
        for (char c : my_string.toCharArray()) {
            if (Character.isDigit(c)) list.add(c - '0');
        }

        return list.stream().sorted().mapToInt(Integer::intValue).toArray();
    }
}

File: 2026-08/src/day_28/prms_120850/Solution.cpp
#include <bits/stdc++.h>
using namespace std;

vector<int> solution(string my_string) {
    vector<int> v;
    for (char c : my_string) {
        if (isdigit(c)) v.push_back(c - '0');
    }
    sort(v.begin(), v.end());

    return v;
}
